lib/routers/profile/pil.js

```diff
@@ -78,7 +79,7 @@ router.param('pil', (req, res, next, id) => {
 
   Promise.resolve()
     .then(() => {
-      return PIL[queryType]().findById(id)
+      return PIL[queryType]().findById(id).eager('establishment')
```

Comment:
This is an issue because it potentially exposes a lot of the information about the PIL holding establishment to anyone who can _view_ the PIL, which is not OK.

Part of the work that was done to originally expose PILs to establishments which don't hold them was under the condition that the holding establishment would _not_ be visible, and this PR makes that information visible.